flytekit/__init__.py

```diff
@@ -13,10 +13,14 @@
 from flytekit.annotated.task import reference_task, task
 from flytekit.annotated.workflow import WorkflowFailurePolicy, reference_workflow, workflow
 from flytekit.loggers import logger
+from flytekit.types import schema
```

Comment:
> we actually have the same problem with the regular pandas library. Tasks and workflows that take in and return pandas dataframes need to trigger the SchemaTransformer, but unless they explicitly import flytekit.types.schema, the transformer isn't registered and it doesn't run. Can you think of a way around this?

@wild-endeavor this is one workaround: importing the relevant types in the top-level `__init__.py` file such that the pandas dataframe transformer is loaded anytime user imports `flytekit`.

As for plugins, see the `import_plugins` function in `plugins/__init__.py`, which is called in this file. It just tries to import all plugins available in the new plugin system. This won't work for third-party flytekit plugins that aren't part of the plugin microlib.

Reply:
ya, for schema we should be able to do this, as we have now made it a requirement